### PR TITLE
e2e: allow to use runc from the system

### PR DESCRIPTION
**Pull request by giuseppe**
a few minor cleanups that I am using locally for running the e2e tests with a different runtime version.

### CODE REVIEW

File: contrib/test/integration/main.yml [hunk 1]
```diff
@@ -21,15 +21,20 @@
...
 
+    - set_fact:
+        force: "{{ force_clone }}"
+
     - name: clone build and install kubernetes
       include: "build/kubernetes.yml"
       vars:
         k8s_git_version: "cri-o-node-e2e-patched-logs"
         k8s_github_fork: "runcom"
+        force_clone: "{{ force }}"
```
Comment (wking) [inline contrib/test/integration/main.yml]: I'm not clear on the format here, so take this with a grain of salt, but going from `force_clone` to `force` above and then from `force` back to `force_clone` here seems confusing.  Is it possible to stick with `force_clone` throughout?

Comment (giuseppe) [reply contrib/test/integration/main.yml]: if you use `force_clone: "{{ force_clone }}"` then Ansible will crash in an endless recursion, as it tries to read its value from the same block.

I am not sure if a set done before this block would help to maintain the same name

Comment (wking) [reply contrib/test/integration/main.yml]: > if you use `force_clone: "{{ force_clone }}"` then Ansible will crash in an endless recursion

Can you just leave `force_clone` off of this `include` (and skip the `set_fact` too) and have it be [inherited][1] through?  I'm not familiar with Ansible's variable inheritance, but from the docs it looks like that might work.

[1]: http://docs.ansible.com/ansible/latest/playbooks_variables.html#variable-precedence-where-should-i-put-a-variable

Comment (giuseppe) [reply contrib/test/integration/main.yml]: I've dropped this patch from the PR for now and see if I can rewrite it in a different way if needed.

**Comment by runcom** [comment]
:+1:

**Pull request merged**